.eslintrc

```diff
@@ -51,6 +52,7 @@ rules:
   no-var: 2
   object-curly-spacing: [2, never]
   one-var: [2, never]
+  operator-linebreak: [2, "after"]
```

Comment:
Carry on, nevermind my comment. As long as there is consistency I'm happy :heart_eyes: :heart_eyes: